### PR TITLE
Add Firefox Voice

### PR DESCRIPTION
**Pull request by ianb**
This adds Firefox Voice to the list of extensions that will be signed

### CODE REVIEW

**Comment by escapewindow** [comment]
Awesome, ty! We'll also need a corresponding patch to [xpi-manifest.yml](https://github.com/mozilla-extensions/xpi-manifest/blob/master/xpi-manifest.yml).

**Comment by ianb** [comment]
@escapewindow I've added xpi-manifest.yml to the commit

File: xpi-manifest.yml [hunk 1]
```diff
@@ -141,3 +141,15 @@ xpis:
       - dist/secure-proxy.xpi
     addon-type: privileged
     install-type: npm
+  - name: firefox-voice
+    description: Firefox Voice
+    additional-emails:
+      - ibicking@mozilla.com
+    repo-prefix: firefox-voice
```
Comment (escapewindow) [inline xpi-manifest.yml:148]: This should match the key of the ci/config.yml dict, which is `firefoxvoice`.

**Comment by escapewindow** [comment]
\o/

**Comment by escapewindow** [comment]
We should now be release-enabled off the `prod` branch.

**Pull request merged**